azerothcore/azerothcore-wotlk
# fix(DB): missing Lunar Festival GOs and creatures

Major Cities are missing Lunar Festival gameobjects and NPCs/Creatures
- Darnassus:  Added the entire Festival location  (gameobjects/creatures)
- Shattrath:  Added both Festival locations  (gameobjects/creatures)
- Dalaran:  Added the entire Festival location  (gameobjects/creatures)
- Silvermoon:  Added the entire Festival location  (gameobjects/creatures)
- Exodar: Added the entire Festival location (gameobjects/creatures)
- Exodar: Removed New Year's event objects from the Lunar Event (Festive 
Kegs, Party Tables, and extra Haybales)

Darnassus missing objects and NPCs mentioned as TODO in issue #4354

Pulled this data from the Trinitycore database.

Note: Shattrath gameobjects had two objects that had GUIDs already assigned
in the Azerothcore database.  I continued on with the guids from the issue #4354 
fix, using 241016 and 241017.





## Changes Proposed:
- Darnassus:  Add the entire Lunar Festival location  (gameobjects/creatures)
- Shattrath:  Add both Lunar Festival locations  (gameobjects/creatures)
- Dalaran:  Add the entire Lunar Festival location  (gameobjects/creatures)
- Silvermoon:  Add the entire Lunar Festival location  (gameobjects/creatures)
- Exodar: Add the entire Lunar Festival location (gameobjects/creatures)
- Exodar: Remove New Year's event objects from the Lunar Event (Festive 
Kegs, Party Tables, and extra Haybales)


## Issues Addressed:
- The Major Cities included in this fix are missing their entire Lunar Festival location.   Shattrath has two locations that are missing. 



## Tests Performed:
- Merged the SQL and visited each location to show the NPCs and gameobjects show up. 
- All locations show up. 




## How to Test the Changes:

- Import the SQL code.
- Visit each of the Cities mentioned in the changes proposed.
- Verify the Lunar Festival Event NPCs and their objects are present.


## Known Issues and TODO List:

- [Exodar's Commoners are duplicated while the Lunar Event is on.  Looking into a fix.  Booty bay has the same problem mentioned in #4330]


## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
